packages/bigtable/src/table.js

```diff
@@ -669,16 +687,27 @@ Table.prototype.insert = function(entries, callback) {
  *     deleted.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {object} callback.apiResponse - The full API response.
+ * @param {object[]} callback.statuses - A status for each entity transaction.
  *
  * @example
  * //-
  * // Insert entities. See {module:bigtable/table#insert}
  * //-
- * var callback = function(err, apiResponse) {
- *   if (!err) {
- *     // Mutations were successful.
+ * var callback = function(err, statuses) {
+ *   if (err) {
+ *     // Error handling omitted.
  *   }
+ *
+ *   // statuses = [
```

Comment:
This object feels redundant. Are the indexes always the same as the index in the array? This seems better:

```js
statuses = [
  { code: 200, message: 'OK' },
  { code: 200, message: 'OK' }
]
```

or:

function (err, insertErrors) {
  insertErrors = [
    {
      index: 21,
      code: 500,
      message: 'Internal Server Error',
      entry: { originalEntryThatFailed }
    }
  ]
}